src/shared/api/mock-factories/block.ts

```diff
@@ -10,3 +11,17 @@ export function makeBlock(block_hash?: string): BlockShallow {
     transactions: randNumber({ min: 1, max: 20 }),
   };
 }
+
+export function makeBlockDetails(height?: number): Block {
+  return {
+    block_hash: hash(64),
+    parent_block_hash: hash(64),
+    rejected_transactions_merkle_root_hash: hash(64),
+    invalidated_blocks_hashes: [],
+    height: height ?? randNumber(),
+    rejected_transactions: [],
+    timestamp: new Date(Date.now() - 10000).toISOString(),
```

Comment:
Suggest using date utils from `@ngneat/falso`, e.g. [`randPastDate`](https://ngneat.github.io/falso/docs/date/#randpastdate)

Reply:
Acknowledged and replaced the timestamp to use ```randPastDate```